packages/server/src/server.ts

```diff
@@ -8,6 +8,10 @@ import { MongoError } from 'mongodb';
 
 dotenv.config();
 
+mongoose.Types.ObjectId.prototype.valueOf = function() {
```

Comment:
why this?

Reply:
GraphQL 14.0.2 issues @jgcmarins https://github.com/graphql/graphql-js/issues/1518